### ISSUE TITLE
need to add optional params to render_token_login_page on reset_password?

### ISSUE THREAD

**Issue by KumanoTanaka** [getredash/redash#3675]
https://github.com/getredash/redash/blob/master/redash/handlers/authentication.py#L85

```python
@routes.route(org_scoped_rule('/reset/<token>'), methods=['GET', 'POST'])
def reset(token, org_slug=None):
    return render_token_login_page("reset.html", org_slug, token)
```

☝️  should be the following?

```python
@routes.route(org_scoped_rule('/reset/<token>'), methods=['GET', 'POST'])
def reset(token, org_slug=None):
    return render_token_login_page("reset.html", org_slug, token, False)
```

In our stand-alone env, we could not reset some user's password.
redash docker tag was: `redash/redash:7.0.0.b18042`

**Comment by rauchy** [comment]
@KumanoTanaka that seems right. Could you submit a PR for this?

**Comment by arikfr** [comment]
What’s the implication of this? I mean what was the behavior before the change?

Arik

**Comment by rauchy** [comment]
@arikfr I'm guessing it will error for pending users. see https://github.com/getredash/redash/blob/master/redash/handlers/authentication.py#L41

**Comment by KumanoTanaka** [comment]
> What’s the implication of this? I mean what was the behavior before the change?

I could not reset user's password when I clicked email-embedded link.
always faced redash error page.
```
This invitation has already been accepted. "
"Please try resetting your password instead.
```

**Comment by arikfr** [comment]
@rauchy oh, ok. I guess this was missing from #3500?

**Comment by woozhijun** [comment]
I have the same problem for redash/redash:7.0.0.b18042

**Comment by justinclift** [comment]
Just hit this exact error when trying to reset a test user's password in a newly installed temporary instance, running Redash 7.0.0 b18042.

The Admin user can generate the "reset password" link, but when going to generated link it turns out to be incorrect:

```
This invitation has already been accepted. 
Please try resetting your password instead.
```

So, resetting passwords this way doesn't work.

The workaround I used, was to connect directly to the backend PostgreSQL database and replace the `password_hash` value for the user (in the `user` table) with the hash for the new password.

**Comment by kouya0219** [comment]
The easiest way 
change `details` in the `users` table

`{"is_invitation_pending": false, "active_at": "2019-09-04T08:53:56Z"}`
to
`{"active_at": "2019-09-04T08:53:56Z"}`

**Comment by fessmage** [comment]
Same problem on Version: 7.0.0+b18042 (4a978bad): reset password don't work, link from email result in page "This invitation has already been accepted".

**Comment by arikfr** [comment]
Closing this as it was already fixed in #3676. The fix is included in `v8.0.0-beta.2` and I recommend upgrading to it to fix it (don't be worried about the `beta` tag, it was in use for along time now).
